src/components/ListView/ListView.js

```diff
@@ -247,6 +247,7 @@ export class ListView extends Component {
                 searchTerm,
                 selectRecord,
                 selectedRecords,
+                resources,
```

Comment:
Do you use this value?

Reply:
Yep, here it is - https://github.com/folio-org/ui-data-import/pull/694/files#diff-eeed0328f61892b3be6ff4abb5f42dc1R71

Reply:
I see, my bad